config/samples/credentials/tower_v1alpha1_ansiblecredential-aws.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: tower.ansible.com/v1alpha1
+kind: AnsibleCredential
+metadata:
+  name: aws-cred
+spec:
+  credential_name: aws-access
+  credential_organization: Default
+  tower_auth_secret: awxaccess
+  credential_description: "This is a test credential"
+  credential_type: "Amazon Web Services"
+  username_secret: aws-secret
```

Comment:
Let's be consistent with whatever awx does here, is it actually client_id and client_secret? Can you check?

Reply:
What you have now matches awx, so we can leave as is.  

* http://pastebin.test.redhat.com/1085623